Sources/Remote Logging/Encrypted Logs/EventLogging+Sentry.swift

```diff
@@ -49,4 +54,16 @@ extension Event {
             return .debug
         }
     }
+
+
+    private static let logIDKey = "logID"
+
+    var logID: String? {
+        get {
+            return self.extra?[Event.logIDKey] as? String
+        }
+        set {
+            self.extra?[Event.logIDKey] = newValue
+        }
+    }
```

Comment:
This used to be in CrashLogging.swift. I moved it closer to where it's actually used. (At one point, I was trying to split remote logging and crash logging into separate modules. They're fairly tightly coupled, though, so I abandoned that for now. But this change was part of that.)